provisioning/plugins/corim-psa-decoder/swcompattributes.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+)
+
+type PSASwCompAttributes struct {
+	MeasurementType  string
+	Version          string
+	SignerID         []byte
+	AlgID            uint64
+	MeasurementValue []byte
+}
+
+func (o *PSASwCompAttributes) FromMeasurement(m comid.Measurement) error {
+	// extract psa-swcomp-id from mkey
+	if !m.Key.IsSet() {
```

Comment:
Question: Is Mkey mandatory in PSA? I thought MKey could be omitted if a SW Component is uniquely identified by a MeasValue and SignerID?

Reply:
The mandatory signerID is carried in an MKey, so MKey can't be optional